File: Facebook List/1 - Two Sum/code.java
/**
 * Status: Accepted
 * 362 / 362 test cases passed
 * Runtime: 99 ms
 * Memory Usage: 39.2 MB
 * Details:
 * - Your runtime is faster than 9.19% of Java online submissions for Two Sum.
 * - Your memory usage are less than 69.54% of Java online submissions for Two Sum.
*/

class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] output = new int[2];
        
        for(int i = 0; i < nums.length; i ++){
            for(int j = 0; j < nums.length; j++){
                if(i != j && nums[i] + nums[j] == target){
                    output[0] = i;
                    output[1] = j;
                    return output;
                }
            }
        }
        
        return null;
    }
}

File: Most Popular Problems (last 6 months)/Facebook/1 - Minimum Remove to Make Valid Parentheses (153 times)/code.js
/**
 * Status: Accepted
 * 62 / 62 test cases passed.
 * Runtime: 123 ms
 * Memory Usage: 52.3 MB
 * Details:
 * - Your runtime is faster than 46.94% of JavaScript online submissions for Minimum Remove to Make Valid Parentheses.
 * - Your memory usage are less than 18.05% of JavaScript online submissions for Minimum Remove to Make Valid Parentheses.
*/ 

/**
 * @param {string} s
 * @return {string}
 */
var minRemoveToMakeValid = function(s) {
    stack = [];
    s = s.split('');
    
    for(let i=0; i < s.length; i++){
        if(s[i] === '(') {
            stack.push({value: '(', index: i})
        }
        else if(s[i] === ')') {
            stack.length == 0 ? s[i] = '' : stack.pop();
        }
    }
    
    while(stack.length > 0) {
        letter = stack.pop();
        s.splice(letter.index, 1)
    }
    
    
    let result = '';

    s.forEach((letter) => {
       result = result + letter;
    });
    
    return result;
};
